addons/hr_holidays/views/hr_leave_allocation_views.xml

```diff
@@ -79,6 +79,7 @@
                     </div>
                     <group>
                         <group>
+                            <field name="is_name_custom" invisible="1"/>
```

Comment:
As I understand, this is not needed anymore following odoo/odoo#137031

Reply:
Actually if I remove it it doesn't work anymore snif

Reply:
Ah ok then pas de souci bébou we can leave it, but we need a comment to explain why in that case :innocent: